circup/__init__.py

```diff
@@ -1311,8 +1319,14 @@ def install(ctx, modules, py, requirement, auto, auto_file):  # pragma: no cover
         with open(requirement, "r") as fp:
             requirements_txt = fp.read()
         requested_installs = libraries_from_requirements(requirements_txt)
-    elif auto:
-        auto_file = os.path.join(ctx.obj["DEVICE_PATH"], auto_file)
+    elif auto or auto_file:
+        if auto_file is None:
+            auto_file = "code.py"
+        if not os.path.isabs(auto_file) and not auto_file[:2] == "./":
```

Comment:
Can you check for `.\` here as well? Maybe `os.path` or Pathlib has some help for this in an OS-independent way.

Reply:
It could use os.sep (and test .. while there)
```suggestion
        is_local = (auto_file[:2] in ("." + os.sep, ".."))
        if not os.path.isabs(auto_file) and not is_local:
```
Or drop it altogether. The idea was to give those paths a meaning, maybe use a local python file as requirements, maybe allow for tab completion (which doesn't seem to work with "./" even if I set the type to click.Path()). So I don't know if it's worth it.